pages/common/gcc.md

```diff
@@ -19,6 +19,6 @@
 
 `gcc -S {{path/to/source.c}}`
 
-- Compile source code without linking:
+- Compile source code into an object file without linking:
```

Comment:
```suggestion
- Compile the source code into an object file without linking:
```

Maybe it's better. Anyway everything looks nice. 👍🏻

Reply:
I don't think it makes a difference. If using "a specific" it would have to be with "source code file".